src/main/resources/schema/sscs-robotics.json

```diff
@@ -97,7 +97,9 @@
           "type": "string",
           "pattern": "^[0-9\\-+ ]{10,17}$"
         },
-        "email": { "type": "string" }
+        "email": { "type": "string" },
+        "isAppointee": {"$ref": "#/definitions/yesOrNo"},
```

Comment:
Is this field needed? I thought this was the one we removed from our data structure (which was put back in to handle old case data)?

Reply:
it's removed now. Thanks.